examples/api/collections.py

```diff
@@ -30,16 +30,14 @@
 theta = np.linspace(0, 2*np.pi, nverts)
 xx = r * np.sin(theta)
 yy = r * np.cos(theta)
-spiral = list(zip(xx, yy))
+spiral = np.column_stack([xx, yy])
 
 # Make some offsets
 # Fixing random state for reproducibility
 rs = np.random.RandomState(19680801)
 
 
```

Comment:
Might as well remove this extra line while you're here.

Reply:
done